tools/linker/RemoveRejectedTypesStep.cs

```diff
@@ -0,0 +1,97 @@
+﻿using System;
+using System.Collections.Generic;
+
+using Mono.Cecil;
+using Mono.Linker;
+using Mono.Tuner;
+using MonoTouch.Tuner;
+using Xamarin.Tuner;
+
+namespace Xamarin.Linker {
+
+	// Important: This is NOT a general purpose code remover. There are a lot of
+	// cases to cover and this implementation cover only the one it requires so far.
+	// IOW additional testing is REQUIRED if you change the scope of the code.
+	public class RemoveRejectedTypesStep : RemoveCodeBase {
+
+		public RemoveRejectedTypesStep ()
+		{
+		}
+
+		public DerivedLinkContext Context {
+			get { return context as DerivedLinkContext; }
+		}
+
+		public override SubStepTargets Targets {
+			get { return SubStepTargets.Assembly; }
+		}
+
+		protected override string Name { get; } = " Removing Rejected Type";
```

Comment:
Extra space at beginning of `Removing Rejected Type`?

Reply:
To late for this, since is a cherry-pick/backport. We should fix that in the correct location :/

Reply:
Good catch :-)
It's only shown on on errors, obviously we hope never to throw unexpectedly here, and won't affect execution (nor would it be a breaking change)